cclib/parser/molcasparser.py

```diff
@@ -0,0 +1,154 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2017, the cclib development team
+#
+# This file is part of cclib (http://cclib.github.io) and is distributed under
+# the terms of the BSD 3-Clause License.
+
+"""Parser for Molcas output files"""
+
+from __future__ import print_function
+
+import re
+
+from cclib.parser import logfileparser
+from cclib.parser import utils
+
+
+class Molcas(logfileparser.Logfile):
+    """A Molcas log file."""
+
+    def __init__(self, *args, **kwargs):
+
+        # Call the __init__ method of the superclass
+        super(Molcas, self).__init__(logname="Molcas", *args, **kwargs)
+
+    def __str__(self):
+        """Return a string representation of the object."""
+        return "Molcas log file %s" % (self.filename)
+
+    def __repr__(self):
+        """Return a representation of the object."""
+        return 'Molcas("%s")' % (self.filename)
+
+    #These are yet to be implemented.
+    def normalisesym(self, label):
```

Comment:
@jevandezande Yes, this still needs to be implemented, much like the rest of the parser. I am still working on it.